common/chains/client/models.go

```diff
@@ -0,0 +1,17 @@
+package client
+
+type SendTxReturnCode int
+
+// Generalized client error types that dictates what should be the next action, depending on the RPC error response.
```

Comment:
I don't think this comment will come through in the godoc. How moving to the type?
```suggestion
// SendTxReturnCode is a generalized client error that dictates what should be the next action, depending on the RPC error 
response.
type SendTxReturnCode int

```